javascript/javascript2/week1/animalName.js

```diff
@@ -0,0 +1,31 @@
+const animalNames = [
+    'Contemplative Teal Beetle',
+    'Unwavering White Ridley',
+    'Sensible Fuchsia Mockingbird',
+    'Steady Plum Boa',
+    'Convivial Purple Cottontail',
+    'Bald Forest Deer',
+    'Silky Lake Owl',
+    'Fat Sea Snake',
+    'Scaled Cave Zebra',
+    'Juicy Glue Gun Kitty'
+];
+
+const spiritAnimal = document.querySelector('.spiritAnimal');
+const button = document.querySelector(".submit");
+button.innerHTML = "submit";
+
+
+function myFunction() {
```

Comment:
Not a descriptive name. 
`generateSpiritName` maybe.

Reply:
yes I forgot to change the name